playground/frontend/lib/modules/examples/models/example_model.dart

```diff
@@ -54,6 +55,7 @@ class ExampleModel with Comparable<ExampleModel> {
     required this.path,
     required this.description,
     required this.type,
+    this.contextLine = 1,
```

Comment:
Nit: Perhaps this will be cleaner to initialize in-line (on line 46).

Reply:
I'm sorry, but I disagree because in this line we only set a default value for the contextLine parameter if we don't get initial. Initially, this parameter sets in the grpc_example_client.dart (line 226).

Reply:
Or maybe I misunderstood your comment(